src/app/core/entity/entity-type-select/entity-type-select.component.html

```diff
@@ -1,11 +1,12 @@
 <mat-form-field>
   <mat-label>{{ label }}</mat-label>
```

Comment:
As per the discussion on Slack, for now we can use `mat-form-field` inside the EntityTypeSelectComponent. As for the approach of splitting `mat-form-field` from the component would require us to implement extends `CustomFormControlDirective` on the `EntityTypeSelectComponent`. For this, we can create the issue for that and implement it later.